packages/labs/motion/src/animate.ts

```diff
@@ -524,6 +551,9 @@ export class Animate extends AsyncDirective {
           fromFrame['transform'] = `${fromFrame['transform'] ?? ''} ${
             op['transform']
           }`;
+          if (op.override !== undefined) {
+            Object.assign(fromFrame, op.override);
```

Comment:
fromFrame and override should probably have more descriptive names here

`fromFrame` is more like, `transformToIntoFromFrame`, and `override` is specifically overriding _that_